README.md

```diff
@@ -192,6 +192,7 @@ Safe Logging can be found at [github.com/palantir/safe-logging](https://github.c
 - `TooManyArguments`: Prefer Interface that take few arguments rather than many.
 - `PreferStaticLoggers`: Prefer static loggers over instance loggers.
 - `LogsafeArgName`: Prevent certain named arguments as being logged as safe. Specify unsafe argument names using `LogsafeArgName:UnsafeArgNames` errorProne flag.
+- `BuilderPrivateConstructor`: Prevent builders with static factory methods from leaking public constructors.
```

Comment:
not 100% sure about the naming conventions we've stuck to, but somehow I feel like flipping this round and naming this after the 'bad' case feels more appropriate to me, so that people could write `@SuppressWarnings("badthing")` to acknowledge there's something dodgy and they're ignoring it?

Perhaps something like `ImplicitPublicBuilderConstructor` or `UnintentionalPublicBuilder` or something?

Reply:
:+1: